astropy/wcs/wcsapi/utils.py

```diff
@@ -75,10 +75,10 @@ def wcs_info_str(wcs):
     # Find largest between header size and value length
     world_dim_width = max(9, len(str(wcs.world_n_dim)))
     world_nam_width = max(
-        9, max(len(x) if x is not None else 0 for x in wcs.world_axis_names)
+        9, *(len(x) if x is not None else 0 for x in wcs.world_axis_names)
     )
     world_typ_width = max(
-        13, max(len(x) if x is not None else 0 for x in wcs.world_axis_physical_types)
+        13, *(len(x) if x is not None else 0 for x in wcs.world_axis_physical_types)
```

Comment:
This could be shortened even more (also at line 78):
```suggestion
        13, *(len(x) for x in wcs.world_axis_physical_types if x is not None)
```

Reply:
Done.

Reply:
This is actually too aggressive at weeding out lengths because world_axis_physical_types is most often None. It was necessary to add a 0 (doesn't effect the answer).

Reply:
Maybe it would be better to catch the `TypeError`?
```python
    try:
        world_typ_width = max(
            13, *(len(x) for x in wcs.world_axis_physical_types if x is not None)
        )
    except TypeError:
        # wcs.world_axis_physical_types was all None
        world_typ_width = 13
```